roles/openshift_logging/README.md

```diff
@@ -12,7 +10,156 @@ generation for Elasticsearch (it uses JKS) as well as openssl to sign certificat
 As part of the installation, it is recommended that you add the Fluentd node selector label
 to the list of persisted [node labels](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-node-host-labels).
 
-### Required vars:
+### Change to variable structure and deployment structure:
+
+Previously, the EFK stack was deployed as either an all-in-one or a split (apps and ops) deployment based on the boolean variable
+`openshift_logging_use_ops`. This has been phased out in favor of being able to configure multiple (currently up to two is supported)
+ES clusters.
+
+To be able to accomplish this, the old variable naming structure has been revamped to follow a dictionary structure as such:
```

Comment:
This is a change to inventory file right?  Do we need to mention possibly the syntax is available as of X version of ansible?

Reply:
Yeah, that'll have to be fully fleshed out. Given that this was/is still just a POC the readme is very high level. I anticipate a lot of documentation required with the change to this new structure.